VideoAPI/VideoApi.UnitTests/Controllers/Tasks/TasksControllerTests.cs

```diff
@@ -68,5 +68,28 @@ public async Task Should_return_not_found_when_no_task_is_found()
             typedResult.StatusCode.Should().Be((int)HttpStatusCode.NotFound);
             commandHandler.Verify(c => c.Handle(It.IsAny<UpdateTaskCommand>()), Times.Once);
         }
+
+        [Test]
+        public async Task Should_return_no_content_on_add_a_task_for_the_participant_in_a_conference()
```

Comment:
I don't think we need to test with mocks when you're testing with the API already in the integration project.

Reply:
removed the tests